org.eclipse.xtend.lib.gwt.test/.project

```diff
@@ -20,6 +20,26 @@
 			<arguments>
 			</arguments>
 		</buildCommand>
+		<buildCommand>
```

Comment:
i am not sure if i like this.
does this require new installed stuff?

Reply:
Do you mean installed into eclipse? No, I have nothing change after the initial install.

This buildCommands were created by "Eclipse  Gradle -> Refresh Gradle Project" and disappear when we use de.esoco.gwt-lib plugin instead of de.esoco.gwt plugin.
The gwt-lib plugin build only the jar for use in an other gwt application. Therefor we can not call "gwtCompile" and that is what we will test: can gwt compile by inheriting "org.eclipse.xtend.lib.Lib".

This is also the difference to the org.wisepersist:gwt-gradle-plugin: now additionaly a war-file will build.
org.wisepersist:gwt-gradle-plugin shows in gradle build that it compile somthing but don't tell in messages about this nor created from the compiled javascript-artifacts a war-file.
Also the from org.wisepersist:gwt-gradle-plugin created jar file don't contain the java-source-file but this necessary if the jar should be used as gwt-lib.